src/admin/components/UserResumeTable/index.tsx

```diff
@@ -13,16 +13,40 @@ interface UserResumeTableProps {
 
 const UserResumeTable: React.FC<UserResumeTableProps> = (props) => {
   const { resumes } = props;
+  const [yearFilters, setYearFilters] = useState([{ text: '2026', value: '2026' }]);
```

Comment:
Can we figure out a better initial value to avoid hardcoding a year like this? By default, we shouldn't have any filters applied to the table when the page loads - how would we represent this in our state?